### ISSUE TITLE
TAPT dataset

### ISSUE THREAD

**Issue by aabid0193** [allenai/dont-stop-pretraining#33]
I am trying to understand the method for TAPT. For chemprot, for example, are you using the same train dataset that is being used for fine-tuning? This chemprot dataset was just augmented with "randomly masking different tokens across epochs, using the masking probability of 0.15". Or is there some other UNLABELED dataset used for chemprot when doing the TAPT, and then this labeled chemprot data that is open sourced was only used for fine tuning on the downstream task?

**Comment by kernelmachine** [comment]
Hi there, sorry about the delay. For TAPT the same dataset for supervised finetuning is used for adaptive pretraining, though you could use a different dataset that's in-distribution to the task too. We technically do this for curated and knn TAPT in the paper, since these involve curating unlabeled data.
